dojo/problem/tasks.py

```diff
@@ -0,0 +1,37 @@
+import json
+import os
+import requests
+from requests.packages.urllib3.exceptions import InsecureRequestWarning
+
+from dojo.celery import app
+from dojo.decorators import dojo_async_task
+from dojo.problem.helper import CONFIG_FILE, validate_json, download_json, save_json_to_cache
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+@dojo_async_task
+@app.task
+def daily_cache_update(**kwargs):
```

Comment:
It's worth it trying to reuse the `load_json()` function here to avoid code duplication.